x/supplier/keeper/claim.go

```diff
@@ -1,64 +1,140 @@
 package keeper
 
 import (
+	"encoding/binary"
+
 	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/pokt-network/poktroll/x/supplier/types"
 )
 
-// SetClaim set a specific claim in the store from its index
-func (k Keeper) SetClaim(ctx sdk.Context, claim types.Claim) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.ClaimKeyPrefix))
-	b := k.cdc.MustMarshal(&claim)
-	store.Set(types.ClaimKey(
-		claim.Index,
-	), b)
+// InsertClaim adds a claim to the store
+func (k Keeper) InsertClaim(ctx sdk.Context, claim types.Claim) {
```

Comment:
NIT: "CreateClaim" and "SubmitProof" have been the nomenclature we've been using most recently. Does it make sense to align with that here if we're going to deviate from the default generated names?